index.css

```
@@ -47,7 +50,49 @@ body {
   user-select: none;
 }
 
-#buttonAction {
+#answer_right {
+  background-color: green;
+  opacity: 0.0;
+  left:0;
+  display: flex;
+  position: absolute;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  min-height: 300px;
+  min-width: 250px;
+  max-width: 100%;
+  border-radius: 50px 0px 0px 50px;
+  user-select: none;
+  cursor: pointer;
+}
+
+#answer_right:hover {
+  opacity: 0.5;
+}
+
+#answer_wrong {
```

```suggestion
#answer__wrong {
```
View above "Naming conventions"